packages/quantic/force-app/main/default/lwc/quanticDocumentSuggestion/quanticDocumentSuggestion.js

```diff
@@ -31,12 +31,16 @@ export default class QuanticDocumentSuggestion extends LightningElement {
    */
   @api engineId;
   /**
-   * Whether or not we want to display the quick view for the document suggestions.
-   * @api
-   * @type {boolean}
-   * @defaultValue `false`
+   * @deprecated
    */
   @api showQuickview = false;
+  /**
+ * Whether or not we want to hide the quick view for the document suggestions.
+ * @api
+ * @type {boolean}
+ * @defaultValue `false`
+ */
+  @api hideQuickview = false;
```

Comment:
Unfortunately, this is introducing a behavior change for an implementation where the customer does not want the quick view to be shown. Since the `show-quickview` attribute would not be specified, the quickview will appear after the upgrade.

This will need to be clearly documented in the release notes so, hopefully, a customer in this situation would know that the `hide-quickview` attribute needs to be added to preserve the desired behavior.

I'm not sure if there is an easy way to make this change non-breaking.